Exercise_C/index.css

```diff
@@ -2,3 +2,30 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
```

Comment:
Great work using flex.

Another way that you could complete this without using using margin.
Increase the height of you container, on the red class you can apply align-self: flex-end or something like that.
OR
Change the html and have two containers.

Consider making a box class
and using id's